src/Mock/mockArticle.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
```

Comment:
The mock folder should contain mock response data in the format that we expect it to come in from our api, not actual react components. Refactor this and rename the folder from `Mock` to `mock`

Reply:
noted.